source/agora/app.d

```diff
@@ -12,12 +20,38 @@
 *******************************************************************************/
 
 module agora.app;
+import agora.common.Config;
+import agora.node.Node;
+
+import vibe.core.args;
+import vibe.core.core;
+import vibe.core.log;
+import vibe.http.router;
+import vibe.http.server;
+import vibe.web.rest;
 
-void main()
+import std.digest.sha;
+import std.stdio;
+import core.stdc.time;
+import core.time;
+
+/// todo: work around this for 'dub test'
+version (unittest) void main () { } else:
+
+/// Application entry point
+public int main (string[] args)
```

Comment:
`private`

Reply:
I can change it if you want but I don't understand why or what for?

Reply:
Just a matter of style I guess. Why would you have it public ? No one should call it.

Reply:
Conceptually it's the entry point of the application. In practice the real main function is in DRuntime.. anyway I'll make it private.